lib/topological_inventory/orchestrator/object_manager.rb

```diff
@@ -115,6 +115,15 @@ def delete_replication_controller(name)
       rescue Kubeclient::ResourceNotFoundError
       end
 
+      def get_collector_image(name)
+        pod = connection.get_pods(
+          :namespace      => my_namespace,
+          :label_selector => "name=topological-inventory-#{name}-operations"
+        ).first
+
+        pod&.spec&.container&.first&.image
```

Comment:
What if this returns `nil` because we have no collector pods running? 

It seems like a potential race condition?

Reply:
Well, it goes off of the running `operations` pod not the currently running collector(s), but nonetheless it is a race condition if the orchestrator were to come up before the operations pod was ready. It would eventually catch up though since the operations pods have to be running in order for any collectors to be of use. 

Basically this change makes the orchestrator dependent on the operations pods. I do handle if the `collector_image` is nil - we skip that source. So it would catch up eventually and spin up the collector.